test/testSetup.ts

```diff
@@ -1,300 +1,165 @@
 /*
- * Copyright (c) 2023, salesforce.com, inc.
+ * Copyright (c) 2024, salesforce.com, inc.
```

Comment:
Was this intentional because you pretty much rewrote the entire testSetup file in this refactor?

Reply:
yes